app/api/user.py

```diff
@@ -298,7 +303,17 @@ def create_user(
         return jsonify(user)
 
     except IntegrityError as error:
-        return jsonify(message=error.message), 409
+        try:
+            msg = error.message
+        except AttributeError:
+            msg = str(error)
+
+        return jsonify(message=msg), 409
 
     except Exception as error:
-        return jsonify(message=error.message), 400
+        try:
+            msg = error.message
+        except AttributeError:
+            msg = str(error)
+
+        return jsonify(message=msg), 400
```

Comment:
return jsonify(message=str(error), 400

Reply:
This looks fine to me for now.  We can think about changing it in the future.